.github/workflows/pull_request_review.yml

```diff
@@ -1,11 +1,39 @@
-name: beta
+name: Pull request review submitted
 on:
   pull_request_review:
     types: [submitted]
 jobs:
-  # upload needed-for-testing build
+  # Return task back to In Progress if pull request is changes requested
+  return_task_to_in_progress:
```

Comment:
Would it not better if we split logic with jira in separated file.
Example:
./.github/beta-deploy.yml
./.github/jira-request-changes.yml
./.github/jira-create-pr.yml

Reply:
Yes of course! Please do it and send Pull request. Thank you